lib/kernel-manager.js

```diff
@@ -81,8 +80,9 @@ export class KernelManager {
     };
 
     const kernel = new ZMQKernel(kernelSpec, grammar, options, () => {
-      store.newKernel(kernel, filePath, editor, grammar);
-      if (onStarted) onStarted(kernel);
+      const kernel_ = new Kernel(kernel);
```

Comment:
Assuming Kernel(=HKernel). This doesn't feel right. Why does ZMQKernel need to know about Kernel(=HKernel)? ZMQKernel only does Jupyter comms.